galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zossecurity.manager/src/main/java/dev/galasa/zossecurity/IZosSecurity.java

```diff
@@ -122,6 +123,7 @@ public interface IZosSecurity {
 	 * Create a new profile on the specified image/sysplex.
 	 * 
 	 * @param image    - The image/sysplex
+	 * @param className 

```

Comment:
why not add some text ?

Reply:
I don't know any meaning to add. So leaving blank.